manifests/u/Ultra/Ultra/1.0.0.4/Ultra.Ultra.yaml

```
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: Ultra.Ultra
+PackageVersion: 1.0.0.4

```

```suggestion
PackageVersion: 5.0.5.0
```
This must match the version in control panel